scripts/build/TestImpactAnalysis/tiaf_persistent_storage.py

```diff
@@ -119,4 +163,16 @@ def has_historic_data(self):
 
     @property
     def last_commit_hash(self):
-        return self._last_commit_hash
\ No newline at end of file
+        return self._last_commit_hash
+
+    @property
+    def is_repeat_sequence(self):
+        return self._last_commit_hash == self._this_commit_hash
+
+    @property
+    def this_commit_last_commit_hash(self):
+        return self._this_commit_hash_last_commit_hash
+
+    @property
+    def can_rerun_sequence(self):
+        return self._has_previous_last_commit_hash
```

Comment:
no newline at end of file (apparently) 🙂

Reply:
Fixed (although in another PR) 👍